trace-utils/src/send_data.rs

```diff
@@ -383,17 +517,410 @@ impl SendData {
 }
 
 #[cfg(test)]
-// For RetryStrategy tests the observed delay should be approximate.
```

Comment:
DId you mean to remove these comments?

Reply:
Nope, a casualty from the rebase. Putting it back.